datafusion/core/tests/sqllogictests/test_files/decimal.slt

```diff
@@ -524,7 +524,7 @@ select * from decimal_simple where c1 >= 0.00004 order by c1 limit 5;
 0.00005 0.000000000005 9 true 0.000052
 
 
-query RRI?R
+query FFIBF
```

Comment:
I updated types using `--complete` in this pr. I pointed datafusion to a local version of `sqllogictest-rs` with changes from this pr https://github.com/risinglightdb/sqllogictest-rs/pull/164. `sqllogictest-rs:0.12` doesn't complete types.